pallets/check-membership/src/loose/mod.rs

```diff
@@ -0,0 +1,64 @@
+//! Pallet that demonstrates a minimal access control check. When a user calls this pallet's
+//! only dispatchable function, `check_membership`, the caller is checked against a set of approved
+//! callers. Only if the caller is approved, do they successfully emit the event.
```

Comment:
The last phrase is a bit hard to understand (for me): "they"? and "the event" – which event is referred to here?

Maybe:
```suggestion
//! callers. If the caller is a member of the set, the pallet's `Event` is emitted, otherwise a `NotAMember` error is returned.
```

Q: Is it ok for callers to assume that "no error" means "is member"? That sounds slightly dangerous as the code could change over time to handle additional error conditions. If you agree maybe we should warn people against using such a pattern? First thing my developer head thought was "so instead of checking that an event was emitted I can just check for `!err`?". Not sure where the proper place for advice on correct coding idioms is though.